### PR TITLE
Test Cases for DataUtilities.calculateRowTotal

### PR DESCRIPTION
**Pull request by bryson189**
Test cases were made for equivalence classes and edge cases.

Please review as some test cases may be overlapping/unnecessary.

### CODE REVIEW

File: Tests/CalculateRowTotalTest.java [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+				will(returnValue(2));
+				allowing(values).getValue(1, 0);
+				will(returnValue(4));
+				allowing(values).getValue(1, 1);
+				will(returnValue(null));
+			}
+		});
+		double result = DataUtilities.calculateRowTotal(values, 1);
+		//Zero should be returned with invalid input
+		assertEquals(result, 0.0, .000000001d);
```
Comment (enochtsang) [inline Tests/CalculateRowTotalTest.java]: What's the `.000000001d`?

File: Tests/CalculateRowTotalTest.java [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+	}
+	
+	@Test
+	public void nullInputTest(){
+		try{
+			double result = DataUtilities.calculateRowTotal(null, 0);
+			assertEquals(result, 0, .000000001d);
+		}catch(NullPointerException e){
+			fail("Expected 0 but null pointer exception caught instead.");
+		}
```
Comment (enochtsang) [inline Tests/CalculateRowTotalTest.java]: There is actually special syntax in JUnit for testing for exceptions, you can look at PR #3 line 88 in CumulativePercentage tests

**Comment by kkcmah** [comment]
I think the expected values and actual values in your assertEquals() calls are switched.
It should be assertEquals(expected, actual).

**Comment by bryson189** [comment]
Expected and actual values have been switched and the testing for exceptions have been changed to match line 88 in PR#3.

**Comment by bryson189** [comment]
Kelby's suggestions have been added: Added default timeout and assertion messages with every test case
